src/main/java/ti4/buttons/ButtonListener.java

```diff
@@ -2800,6 +2805,7 @@ public void resolveButtonInteractionEvent(ButtonContext context) {
                     ButtonHelper.deleteMessage(event);
                 }
                 case "acquireATech" -> ButtonHelper.acquireATech(player, game, event, messageID, false);
+                case "acquireAUnitTechWithInf" -> ButtonHelper.acquireATech(player, game, event, messageID, false, Set.of(Constants.UNIT), "inf");
```

Comment:
We're trying to move away from adding to ButtonListener and using @ButtonHandler notation instead, although I do see a bunch of params here that won't get passed through the ButtonHandler. You can either leave it as is or create a wrapper method over ButtonHelper.acquireATech to pass the messageID, false, Set, & "inf" in